frontend/src/pages/Player/MetadataBox/MetadataBox.tsx

```diff
@@ -189,12 +189,16 @@ export const MetadataBox = React.memo(() => {
 		const displayName = getDisplayName(session)
 		const userParam = validateEmail(displayName) ? 'email' : 'identifier'
 
-		setSearchQuery((query) =>
-			buildQueryStateString({
+		setSearchQuery((query) => {
+			const params = session.identified
+				? { [`user_${userParam}`]: displayName }
```

Comment:
Just wanted to check if you knew of the `getDisplayNameAndField` method? Seems to have the different cases listed with the value and identifier.

Reply:
Thanks for the heads up!

I've reviewed the function. It looks like it's doing a fall-through where it looks for name, email, identifier or fingerprint. I think this change still checks out, because the fingerprint should be universally available and maps to the `session_device_id`.